app/page/template/content/conversation/input-bar.htm

```diff
@@ -42,7 +42,7 @@
             </gif-icon>
           <!-- /ko -->
 
-          <!-- ko ifnot: input().length -->
+          <!-- ko ifnot: input().text.length -->
```

Comment:
Does this actually work? Text is no longer an observable. Changes to it should remain undetected by knockout.

Reply:
But `input()` still is an observable. Before `input()` just returned a string, now it returns an object.

Reply:
And yes, it works :slightly_smiling_face:

Reply:
Weird. Because the object rarely changes. It stays the same object and only parts of it change.